docs/en/reference/pipeline-integrations/toolkit-apps/tk-multi-breakdown.md

```diff
@@ -7,19 +7,36 @@ lang: en
 
 # Breakdown
 
-The Scene Breakdown app shows you a list of referenced content and tells you which items are out of date. You access it from the {% include product %} Menu.
+If you are using {% include product %} for managing your Published Files, use the Scene Breakdown app to keep your references up to date. The Scene Breakdown app helps you manage references and versioning of {% include product %} Published Files. The breakdown app allows you to:
 
-![Overview of the breakdown](../images/apps/multi-breakdown-breakdown.png)
+- Manage references in multiple {% include product %} Projects
+- Override to a version of a PublishedFile that is not the latest
+- Get version history file information
+- Multiple configurable and scalable viewing styles available
 
-You can select one more more items and hit update selected. This will switch the items to use the latest version of the content.
+![Overview of the breakdown2 app](../images/apps/07_scenebreakdown1.png)
 
-## Scanning the Scene
+Please refer to the following documentation for the latest breakdown app:
 
-When the breakdown app scans the scene for references, it will call a special scanning hook. The scanning hook will return a list of nodes with file corresponding paths. For each reference it finds, it returns the file path, and tank will look at the file path and first check if it recognizes it as a publish, and if it does, see if there is a more recent version available.
+- [Breakdown App User Documentation](https://help.autodesk.com/view/SGSUB/ENU/?guid=VRED_ShotGrid_for_Automotive_Design_VRED_ShotGrid_Worflows_Scene_Breakdown_in_VRED_html)
+- [Breakdown App Developer API Documentation](https://developer.shotgridsoftware.com/tk-multi-breakdown2/)
+- [Release Notes](https://github.com/shotgunsoftware/tk-multi-breakdown2/wiki/Release-Notes)
+
```

Comment:
I would love to see a horizontal line to divide the page, other than that looks great - thanks!

Reply:
I like it - do you agree @shaynacohen ? Thanks for adding in in there!

Reply:
Yes! It looks slick. Thanks for the suggestion! I was not expecting it to work the first time (I was experimenting). Let me know when this is good to publish, and I can merge!